### ISSUE TITLE
Registister new claims at IANA

### ISSUE THREAD

**Issue by beusink** [Logius-standaarden/OIDC-NLGOV#10]
To increase interoperability, please register at IANA so that the newly devised claims get on this list: https://www.iana.org/assignments/jwt/jwt.xhtml

See the note on https://www.iana.org/assignments/jwt/jwt.xhtml on registering.

**Comment by TheBonheurs** [comment]
@beusink do you have specific claims in mind or is it a general remark for any future claims we might add?

For the team, what custom claims do we want to append to this list?
- OIN?
- ...

**Comment by beusink** [comment]
Basically any newly introduced claim not already on the IANA list (or an [IETF RFC](https://www.rfc-editor.org/)).

claims like:
- alt_sub
- sub_id_type
- represents 
- sub_id_types_supported

**Comment by hhudig** [comment]
Good point for the _OIN_ claim. For the claims _represents_, _alt_sub_ and _sub_id_type_ we are considering to align them with rfc9396 and/or rfc8693 (rfc9396).  In that context the claims would become standard claims from rfc8693 or would become claims as part of  the type of an Authorization Request. Depending on the choices we may or may not need to register (all of) them at IANA.

**Comment by sanderke** [comment]
Related to
- #12
